pyproject.toml

```diff
@@ -8,12 +8,15 @@ readme = "README.md"
 repository = "https://github.com/TechnologyBrewery/kappa-maki"
 
 [tool.poetry.dependencies]
-python = "^3.11"
+python = ">=3.8"
 
 [tool.poetry.dev-dependencies]
 black = ">=24.3.0"
 behave = ">=1.2.6"
 
+[tool.poetry.group.dev.dependencies]
+pylint = "^3.1.0"
```

Comment:
A: Switch to ">=3.1.0"

Reply:
I'd advocate for keeping it as `^` since it's a dev dependencies that isn't exported as a requirement for users of the package.  So the niceties of keeping the version pegged to avoid breaking changes doesn't come with the "supported-versions" maintenance downsides you'd see from a standard dependencies.

Reply:
✅ Good catch - will create a ticket in Habushu to fix this at the source as it added it in here.